### PR TITLE
NMS-8551: Notifd configuration file

### PR DESCRIPTION
**Pull request by indigo423**
Describe which configuration files are related to Notifd and if changes require restart, reload or get immediately applied.

* JIRA: http://issues.opennms.org/browse/NMS-8551
* Bamboo:

### CODE REVIEW

**Comment by agalue** [comment]
There are other configuration files related only with Notifd:

- notificationCommands.xml
- destinationPaths.xml

The following files are also related with Notifd:

- users.xml
- groups.xml

I'm almost 100% sure that all the above files are reloaded automatically when they are modified.

**Comment by indigo423** [comment]
@agalue 👍 good one

**Comment by indigo423** [comment]
@agalue added users.xml and groups.xml

File: opennms-doc/guide-admin/src/asciidoc/text/operation/daemon-config-files/notifd.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../../images
+
+[[ga-opennms-operation-daemon-config-files-pollerd]]
```
Comment (j-white) [inline opennms-doc/guide-admin/src/asciidoc/text/operation/daemon-config-files/notifd.adoc]: pollerd vs notifd

File: opennms-doc/guide-admin/src/asciidoc/text/operation/daemon-config-files/notifd.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+==== Notifd
+
+[options="header, autowidth"]
+|===
+| Internal Daemon Name | Reload Event
+| _Notifd_            | `uei.opennms.org/internal/reloadDaemonConfig -p 'daemonName Notifd'`
+|===
+
+.Pollerd configuration file overview
```
Comment (j-white) [inline opennms-doc/guide-admin/src/asciidoc/text/operation/daemon-config-files/notifd.adoc]: pollerd vs notifd

File: opennms-doc/guide-admin/src/asciidoc/text/operation/daemon-config-files/notifd.adoc (hunk 1 above)
Comment (j-white) [inline opennms-doc/guide-admin/src/asciidoc/text/operation/daemon-config-files/notifd.adoc]: I'm guessing you may of missed this one when you fixed the other one :)

**Pull request merged**